### PR TITLE
doc: add security policy

### PR DESCRIPTION
**Pull request by patricklodder**
Opening this draft while awaiting other maintainers' details.

Addresses #3556

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+## Reporting a Vulnerability
+
+To report security issues, please send pgp encrypted email to the following
+developers:
+
+
+| Name           | E-mail                  | PGP key fingerprint
+| :------------- | :---------------------- | :-------------------
+| Patrick Lodder | patricklodder@proton.me | E324 9D4E E7F3 7C7F 8B30 676F 603E F476 B796 1EA7
+
```
Comment (michilumin) [inline SECURITY.md:16]: `| Michi Lumin | michi@luskwood.org | 6DB4 66B1 7BD2 60F4 4950 F6AF 520B 53AE 29E9 8861`

**Comment by michilumin** [comment]
Pinged @chromatic to take a look at this as well.

File: SECURITY.md (hunk 1 above)
Comment (patricklodder) [reply SECURITY.md:17]: ✅

**Comment by chromatic** [comment]
email: chromatic@wgz.org
fingerprint: E343 7979 A840 E43F 7BB5 A579 2C48 E041 36C2 B580

**Comment by patricklodder** [comment]
Squashed with all info there. [Pre-squash tree](https://github.com/patricklodder/dogecoin/tree/doc/security-presquash).

Now ready for review

**Review by michilumin — APPROVED** [review]
looks good, and fast responsive action so 👍

**Comment by patricklodder** [comment]
@chromatic - would you mind giving this a final look? I'd still like to be able to get info to see if we need to release a 1.14.8 with haste.

**Review by chromatic — APPROVED** [review]
+1 from me

**Pull request merged**